src/site/paragraphs/lists_of_links.drupal.liquid

```diff
@@ -2,31 +2,100 @@
-  {% for vaParagraph in entity.fieldVaParagraphs %}
-    {% assign hasEvenIndex = forloop.index | modulo: 2 %}
-    <div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-l-col--5{% if hasEvenIndex %} medium-screen:vads-u-margin-right--8{% endif %}">
-      <!-- Section header -->
-      <h3>{{ vaParagraph.entity.fieldSectionHeader }}</h3>
-
-      <ul class="usa-unstyled-list">
-        <!-- Links -->
-        {% for fieldLink in vaParagraph.entity.fieldLinks %}
-          <li class="vads-u-padding-y--1">
-            <a href="{{ fieldLink.url.path }}" rel="noreferrer noopener">
-              {{ fieldLink.title }}
-            </a>
-          </li>
-        {% endfor %}
-
-        <!-- See more articles link -->
-        {% if vaParagraph.entity.fieldLink.url.path %}
-          <li class="vads-u-padding-y--1">
-            <a class="vads-u-text-decoration--none" href="{{ vaParagraph.entity.fieldLink.url.path }}" rel="noreferrer noopener">
-              {{ vaParagraph.entity.fieldLink.title }} <i class="fa fa-chevron-right vads-u-padding-left--0p5 vads-u-font-size--sm" aria-hidden="true" role="presentation"></i>
-            </a>
-          </li>
-        {% endif %}
-      </ul>
-    </div>
+  {% assign formattedVaParagraphs = entity.fieldVaParagraphs | formatVaParagraphs %}
+
+  <!-- Categories -->
+  {% for vaParagraph in formattedVaParagraphs %}
+    {% if forloop.index <= 6 %}
+      <div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-l-col--3 medium-screen:vads-u-margin-right--8">
+        <!-- Section header -->
+        <h3>{{ vaParagraph.entity.fieldSectionHeader }}</h3>
+
+        <ul class="usa-unstyled-list">
+          <!-- Links -->
+          {% for fieldLink in vaParagraph.entity.fieldLinks %}
+            <li class="vads-u-padding-y--1">
+              <a href="{{ fieldLink.url.path }}" rel="noreferrer noopener">
+                {{ fieldLink.title }}
+              </a>
+            </li>
+          {% endfor %}
+
+          <!-- See more articles link -->
+          {% if vaParagraph.entity.fieldLink.url.path %}
+            <li class="vads-u-padding-y--1">
+              <a class="vads-u-text-decoration--none" href="{{ vaParagraph.entity.fieldLink.url.path }}" rel="noreferrer noopener">
+                {{ vaParagraph.entity.fieldLink.title }} <i class="fa fa-chevron-right vads-u-padding-left--0p5 vads-u-font-size--sm" aria-hidden="true" role="presentation"></i>
+              </a>
+            </li>
+          {% endif %}
+        </ul>
+      </div>
+    {% endif %}
   {% endfor %}
+
+  <!-- Extra Categories -->
+  <div aria-hidden="true" class="vads-u-display--none vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row medium-screen:vads-u-flex-wrap--wrap" data-hidden-lists_of_links-entity-id="{{ entity.entityId }}">
```

Comment:
Do we need to check `formattedVaParagraphs.length > 6` outside this div?

Reply:
Not sure if I understand your question 🙂